python/dask_cudf/dask_cudf/core.py

```diff
@@ -23,7 +23,7 @@
 import cudf
 import cudf._lib as libcudf
 
-from dask_cudf import batcher_sortnet, join_impl
```

Comment:
Does this mean that we can remove the `join_impl.py` file as well?

Reply:
Right, sorry - We should be able to remove `join_impl.py`